packages/react-native-session-replay/ios/Sources/DdSessionReplayImplementation.swift

```diff
@@ -60,9 +65,30 @@ public class DdSessionReplayImplementation: NSObject {
             startRecordingImmediately: startRecordingImmediately,
             customEndpoint: customEndpointURL
         )
-                    
+        
+        let bundle = Bundle(for: DdSessionReplayImplementation.self)
+        var svgMap: [String: SVGData] = [:]
+        
+        if let url = bundle.url(forResource: "assets", withExtension: "json") {
+            do {
+                let data = try Data(contentsOf: url)
+                let decoder = JSONDecoder()
+                svgMap = try decoder.decode([String: SVGData].self, from: data)
+            } catch {
+                consolePrint("Failed to load or decode assets.json", .debug)
```

Comment:
Shouldn't we stop here when it fails to load thge assets? We can also add the error to help troubleshooting.

```suggestion
                consolePrint("Failed to load or decode assets.json: \(error)", .debug)
                return
```